src/components/ItemList/index.js

```diff
@@ -59,9 +61,123 @@ export function ItemList({ items }) {
     setOpenItem(items[index])
     setOpenPrice(prev => !prev)
   }
-
+  function unauthhandleClickPrice() {
+    setOpenPrice(prev => !prev)
+  }
+  function handleClickDelete(item) {
+    var result = fetch(`${API_BASE_URL}/items/${item._id}`, {
+      method: 'DELETE',
```

Comment:
```suggestion
      method: 'DELETE',
      headers: { authorization: 'Bearer ' + localStorage.getItem(key) },
```
We also need to pass the auth token for delete an item api.
Sorry for catching this late.